java/src/jmri/Block.java

```diff
@@ -135,7 +135,7 @@ public Block(String systemName, String userName) {
     static final public int TIGHT = 0x02;
     static final public int SEVERE = 0x04;
 
-    // this should only be used for debugging…
+    // this should only be used for debugging
```

Comment:
The use of non-ascii characters (though in comments) cause serious problems, at least in IntelliJ IDE.
I've run through the code and removed/replaced them. Note to @geowar1